processing/src/test/java/org/apache/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -3700,7 +3702,7 @@ public void testGroupByWithOrderOnHyperUnique()
             query,
             "1970-01-01T00:00:00.000Z",
             "market",
-            "upfront",
+            "total_market",
```

Comment:
Are these changes from the stabilization?

Reply:
Yes.